packages/console/src/components/RadioGroup/Radio.tsx

```diff
@@ -18,7 +19,7 @@ const Check = () => (
 export type Props = {
   className?: string;
   value: string;
-  title?: AdminConsoleKey;
+  title?: AdminConsoleKey | ReactElement<typeof DangerousRaw>;
```

Comment:
nit: better give different props to represent the DangerousRaw typed title?

Reply:
This pattern was decided in a previous discussion and is now being followed in the AC.